kolibri/plugins/coach/api.py

```diff
@@ -134,11 +135,16 @@ def get_queryset(self):
 
         if classroom_id:
             try:
-                Collection.objects.get(pk=classroom_id)
+                collection = Collection.objects.get(pk=classroom_id)
```

Comment:
Maybe rename the variables `classroom_id` within this method to `collection_id`, so we know it is more general now.

Reply:
And maybe add a comment explaining that `classroom_id` in LearnerProgressNotification model is a misnomer now, since it could refer to a learnergroup

Reply:
Totally agree. Done